2_run.R

```diff
@@ -0,0 +1,40 @@
+source('2_run/src/run_glm3.R')
+
+p2 <- list(
+  # Function will generate file
+  # but return a tibble that includes that filename and its hash
+  tar_target(
+    p2_glm_uncalibrated_runs,
+    run_glm3_model(
+      sim_dir = '2_run/tmp',
+      nml_objs = p1_nml_objects,
+      meteo_xwalk = p1_meteo_xwalk,
+      export_fl_template = '2_run/tmp/%s_%s_%s.feather'),
+    packages = c('retry','glmtools', 'GLM3r'),
+    pattern = map(p1_meteo_xwalk)),
+  # Group model runs by lake id and gcm
+  tar_target(
+    p2_glm_uncalibrated_run_groups,
+    p2_glm_uncalibrated_runs %>% 
+      group_by(lake_id, gcm) %>% 
+      tar_group(),
+    iteration = "group"
+  ),
+  # Use grouped target to combine glm output into feather files
+  tar_target(
+    p2_glm_uncalibrated_output_feathers,{  
+      # set filename
+      outfile <- sprintf('2_run/out/%s_%s.feather',
+                         unique(p2_glm_uncalibrated_run_groups$lake_id),
+                         unique(p2_glm_uncalibrated_run_groups$gcm))
+      if(!dir.exists(dirname(outfile))) dir.create(dirname(outfile), recursive=TRUE)
```

Comment:
I think I've settled on skipping this type of code and committing an empty file in the directories that I everyone needs. So, committing `2_run/out/.empty`. [Here is an example](https://code.usgs.gov/wma/proxies/habs/wq-data-prep/-/tree/main/1_fetch/out) of this behavior. That way, the pipeline will run and you won't need to add checks everywhere for whether the directories exist.